internal/btf/btf.go

```diff
@@ -94,6 +90,29 @@ func LoadSpecFromReader(rd io.ReaderAt) (*Spec, error) {
 	return loadSpecFromELF(file)
 }
 
+// LoadSpecAndExtInfosFromReader reads from an ELF.
+//
+// ExtInfos may be nil if the ELF doesn't contain section metadta.
+// Returns ErrNotFound if the ELF contains no BTF.
+func LoadSpecAndExtInfosFromReader(rd io.ReaderAt) (*Spec, *ExtInfos, error) {
```

Comment:
Following the `ExtInfos{Spec}` relationship you suggested, following suit would make sense:

```suggestion
func LoadExtInfosFromReader(rd io.ReaderAt, spec *Spec) (*ExtInfos, error) {
```

Reply:
That was my initial plan, but this would allow using a Spec loaded from a different ELF.